conda_smithy/cli.py

```diff
@@ -94,8 +73,6 @@ def __init__(self, parser):
         scp.add_argument("--feedstock-directory", default='./{package.name}-feedstock',
                          help="Target directory, where the new feedstock git repository should be "
                          "created. (Default: './<packagename>-feedstock')")
-        scp.add_argument("-m", "--variant-config-files", action="append",
```

Comment:
Why remove this?  It doesn't hurt to give people the option?  I don't expect it will get used much, so it's OK with me to remove it.  I just want it to be something that everyone agrees should not be used.

Reply:
So I had asked @isuruf to remove this. It certainly makes sense to have such an option with `conda-build` undoubtedly. Though saw less value in having this option for a feedstock for the following reasons.

1. This is already possible by including a config file in the `recipe`
2. Re-rendering with config files outside the feedstock complicates reproducibility
    (particularly for others)
3. It's unclear at this time that such an option is useful for feedstocks

Completely agree that it is reasonable to provide people options. Though would like to make sure they are not redundant, are reproducible by others, and have motivating use cases. If any of these change, certainly happy to revisit. Also if I'm missing something, please feel free to correct me.

Reply:
This is reasonable.  I'm OK with removing it here.  It does make much more sense as an "experimental" thing to do when playing locally with conda-build, and that's not really the scope of conda-smithy.